Assets/Packages/com.studio23.ss2.dialoguesystem/Runtime/Data/Models/CharacterData.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Studio23.SS2.DialogueSystem.Data
+{
+    [CreateAssetMenu(menuName = "Studio-23/Dialogue System/New Character Data")]
+    [Serializable]
+    public class CharacterData : ScriptableObject
+    {
+        public string CharacterID;
+        public string CharacterName;
+        public Color DialogueColor;
+        public List<CharacterExpressionImage> ExpressionTable;
+    }
+
+    [Serializable]
+    public class CharacterExpressionImage
+    {
+        public string Reaction;
```

Comment:
as it's public repo. I'd rather insist on keeping string instead of hardcoding enums. Although we could provide a static string class generation method perhaps but that seems a bit over kill